docs/content/features/run_on_aws.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Infection Monkey AWS Execution"
+draft: false
+description: "Run Infection Monkey on AWS EC2 instances"
+tags: ["aws", "ec2", "ssm", "agent", "run"]
+pre: "<i class='fa-brands fa-aws'></i> "
+---
+
+## Description
+
+The Infection Monkey can run on AWS EC2 instances using the
+[AWS Systems Manager (SSM) Agent](
+https://docs.aws.amazon.com/systems-manager/latest/userguide/ssm-agent.html).
+Each EC2 instance has an SSM Agent installed for updating, managing, and
+configuring the instance. It executes requests received from
+the [AWS Systems Manager](
+https://docs.aws.amazon.com/systems-manager/latest/userguide/what-is-systems-manager.html),
+and uses the [Amazon Message Delivery Service](
+https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazonmessagedeliveryservice.html)
+to report back status and execution details.
+
+Infection Monkey AWS Execution lets you run Infection Monkey Agent on your
+chosen EC2 instances without the requirement of manual installation, enabling
+the simulation of various attack scenarios across your AWS infrastructure.
```

Comment:
Move the last paragraph up towards the beginning and start this section with something like, "Infection Monkey can be run on AWS EC2 instances without any manual installation required, enabling the simulation of various attack scenarios across your AWS infrastructure," or, "Infection Monkey enables the simulation of various attack scenarios across your AWS infrastructure without the requirement of any manual installation."

I prefer the second. It's enough for the introduction and other details will be explained in the sections ahead and in the how-to.